app/src/main/java/com/babylon/wallet/android/presentation/ui/composables/DefaultSettingsItem.kt

```diff
@@ -42,7 +42,12 @@ fun DefaultSettingsItem(
         icon?.let {
             Icon(painter = painterResource(id = it), contentDescription = null)
         }
-        Column(modifier = Modifier.height(IntrinsicSize.Max), verticalArrangement = Arrangement.Center) {
+        Column(
+            modifier = Modifier
+                .weight(10f)
```

Comment:
Didn't come up with anything better to not make chevron hide when subtitle goes two line.. Open for suggestions.